src/transformers/data/metrics/__init__.py

```diff
@@ -60,7 +60,8 @@ def pearson_and_spearman(preds, labels):
 def glue_compute_metrics(task_name, preds, labels):
     warnings.warn(DEPRECATION_WARNING, FutureWarning)
     requires_backends(glue_compute_metrics, "sklearn")
-    assert len(preds) == len(labels), f"Predictions and labels have mismatched lengths {len(preds)} and {len(labels)}"
```

Comment:
Please don't touch this file as it is deprecated and we are not accepting any new PRs modifying it.